addons/medical_treatment/functions/fnc_onMedicationUsage.sqf

```diff
@@ -18,15 +18,15 @@
  * Public: No
  */
 
-params ["_target", "_className", "_maxDosage", "_incompatibleMedication"];
-TRACE_4("onMedicationUsage",_target,_className,_maxDosage,_incompatibleMedication);
+params ["_target", "_className", "_maxDose", "_chanceDoses", "_incompatibleMedication"];
+TRACE_5("onMedicationUsage",_target,_className,_maxDose,_chanceDoses,_incompatibleMedication);
 
 private _overdosedMedications = [];
 
 // Check for overdose from current medication
-if (_maxDosage > 0) then {
+if (_maxDose > 0) then {
     private _currentDose = [_target, _className] call EFUNC(medical_status,getMedicationCount);
-    if (_currentDose >= floor (_maxDosage + round(random(2)))) then {
+    if (_currentDose > floor (_maxDose + round(random(_chanceDoses)))) then {
```

Comment:
```suggestion
    if (_currentDose > _maxDose + floor(random(_chanceDoses + 1))) then {
```
Personally since we're introducing configuration for this variation I think we should make the distribution uniform so it's easy to reason about.

Alternatively we could do `random [0, 0, _changeDoses + 1]` for a more normal distribution (tends towards 0, but can reach the higher dosage too).

Reply:
Uniform is the way to go, I agree.